### PR TITLE
rosmobile_build_tools: 0.4.2-1 in 'rolling/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by RonaldsonBellande**
Increasing version of package(s) in repository `rosmobile_build_tools` to `0.4.2-1`:

- upstream repository: git@github.com:Application-UI-UX/rosmobile_build_tools.git
- release repository: git@github.com:Application-UI-UX/rosmobile_build_tools.git
- distro file: `rolling/distribution.yaml`
- bloom version: `0.11.2`
- previous version for package: `null`

## rosmobile_build_tools

```
* Fix countless bugs in the repository and recalibrate
* Release dedicated code for maven, ros, and python
* Maintainer & Contributors: Ronaldson Bellande
```

### CODE REVIEW

File: rolling/distribution.yaml [hunk 1]
```diff
@@ -5529,6 +5529,21 @@ repositories:
...
+      url: https://github.com/Application-UI-UX/rosmobile_build_tools/blob/main/README.md.git
+      version: main
+    release:
+      tags:
+        release: release/rolling/{package}/{version}
+      url: git@github.com:Application-UI-UX/rosmobile_build_tools.git
+      version: 0.4.2-1
+    source:
+      type: git
+      url: https://github.com/Application-UI-UX/rosmobile_build_tools/blob/main/README.md.git
```
Comment (clalancette) [inline rolling/distribution.yaml:5544]: This must point to the base of the repository, and end in `.git`.  Thus this needs to be `https://github.com/Application-UI-UX/rosmobile_build_tools.git`.

**Review by clalancette — CHANGES_REQUESTED** [review]

File: rolling/distribution.yaml [hunk 2]
```diff
@@ -5529,6 +5529,21 @@ repositories:
...
     status: developed
+  rosmobile_build_tools:
+    doc:
+      type: git
+      url: https://github.com/Application-UI-UX/rosmobile_build_tools/blob/main/README.md.git
+      version: main
+    release:
+      tags:
+        release: release/rolling/{package}/{version}
+      url: git@github.com:Application-UI-UX/rosmobile_build_tools.git
```
[the remark this answers is not in the document]
Comment (RonaldsonBellande) [reply rolling/distribution.yaml:5540]: Yes, I will wait until I get the instructions and full review.

File: rolling/distribution.yaml (hunk 1 above)
Comment (RonaldsonBellande) [reply rolling/distribution.yaml:5544]: yea, my fault. I saw doc and source and though hmmm, give the README.md location, even tho other package I published(this package as well) did not need such things.

**Review by tfoote — CHANGES_REQUESTED** [review]
I'd like to reinforce @clalancette 's comment that this package doesn't meet our naming conventions. From [REP 144](https://www.ros.org/reps/rep-0144.html) 

It shouldn't be using `ros` as the prefix. And then `mobile_build_tools` is too generic. It doesn't represent all potential build tools for mobile platforms. The best would be for it to have a specific name.

**Pull request closed without merging**